src/utils.tsx

```
@@ -0,0 +1,10 @@
+import { PreviewList, previewList } from "./models";
+
+export function fromJsonToProp(jsonApi: JSON): PreviewList {
+  const propList = previewList.safeParse(jsonApi);
+  if (propList.success) {
+    return propList.data;
+  } else {
+    return { businesses: [] } as PreviewList;
```

If this parsing fails, it should be handled as an error, instead of failing silently.